src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-security-lake-monitoring-integration.mdx

```diff
@@ -7,37 +7,108 @@ tags:
+[New Relic infrastructure integrations](/docs/infrastructure/integrations-getting-started/getting-started/introduction-infrastructure-integrations) include an integration for AWS Security Lake, allowing you to send your security log data to New Relic. 
 
 Collect and send telemetry data to New Relic from [Security Lake](https://aws.amazon.com/security-lake/) using our integration. You can use this integration to monitor your services, query incoming data, and build dashboards to observe everything at a glance.
 
 ## Activate integration [#activate]
 
-To enable this integration, set up S3 log forwarder. We suggest using our application for ease and convenience, but you can also set your own.
-
-## Configuration 
-
-Install our dedicated [Amazon Security Lake log forwarder](/docs/logs/forward-logs/aws-lambda-sending-security-logs-s3/) but instead of Lambda trigger, use AWS SQS service. Set up variables accordingly.
-
-## Logs
-
-To find your logs on New Relic, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Logs** and set **Attributes** to `product.name`, choose one or few of your interest.
-
-The followings source types of logs are currently supported:
+To enable this integration, set up an S3 log forwarder. We suggest using our application for ease and convenience, but you can also set your own.
+
+## Configuration steps [#steps]
+
+Here's an overview of the steps you'll be doing: 
+
+1. [Create a Security Lake subscriber](#createSubscriber).
+2. [Install our dedicated Amazon Security Lake log forwarder](#installLogForwarder).
+3. [Find and use your log data](#logs).
+
+## Create Security Lake subscriber [#create-subscriber]
+
+1. Navigate to the Security Lake feature in your [AWS Console](console.aws.amazon.com/securitylake).
+2. Select **Subscribers**, and select **Create Subscriber**.
+3. Give the subscriber a name and select a region.
+4. Select which log and event sources you want sent to New Relic.
+4. Fill out the remaining details as described below:
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+      Field 
+      </th>
+      <th>
+      Value  
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+      `Data access method`
+      </td>
+      <td>
+      S3  
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `Account ID`  
+      </td>
+      <td>
+      Insert the AWS `account ID` for the account that you want to set up monitoring for
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `External ID`
+      </td>
+      <td>
+      Insert your New Relic `AccountID`
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `Notification details`
+      </td>
+      <td>
+      SQS queue  
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+5. Select **Create**.
+
+Your `SecurityLakeSQSArn` and `SecurityLakeSubscriberRoleArn` are generated here. Take note of them because you'll need them for the next step.
```

Comment:
There are no fields like this in AWS UI. You need to explain how to correlate output in UI with what is expected in function paramters